Explorer/Assets/DCL/Audio/Systems/CalculateOceanAudioStatesJob.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DCL.Audio.Systems;
+using System.Numerics;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Jobs;
+using Unity.Mathematics;
+
+namespace DCL.Audio.Jobs
+{
+    [BurstCompile]
+    public struct CalculateOceanAudioStatesJob : IJobParallelFor
+    {
+        private NativeArray<OceanAudioState> oceanAudioStates;
+        private readonly UnityEngine.Vector3 cameraPosition;
```

Comment:
You should be using `float3` in Jobs

Reply:
good catch, missed it when I copied the code and let rider do some auto-magicks xD